check_client_query_compiler_methods.py

```diff
@@ -0,0 +1,33 @@
+import inspect
```

Comment:
What's the purpose of this file?

Reply:
This is how I check that I've implemented the all the client query compiler methods that I want to implement. We should keep it around in case we want to do the check again (maybe for every PR on this branch).